custom/icds_reports/tasks.py

```diff
@@ -538,6 +534,8 @@ def _run_custom_sql_script(commands, day=None, db_alias=None):
 @track_time
 def aggregate_awc_daily(day):
     with transaction.atomic(using=db_for_read_write(AggAwcDaily)):
+        AggAwcDaily.aggregate(force_to_date(day) - timedelta(days=2))
```

Comment:
added these here, so that its not on its caller to call thrice, trying to exclude the chance of forgetting to run it thrice with different dates as the caller changes

Reply:
can you make them independently atomic?

Reply:
@Rohit25negi this is an important suggestion, please submit a follow up PR

Reply:
@snopoke  sorry. I missed your comment. Sure, sending followup Pr.

Reply:
https://github.com/dimagi/commcare-hq/pull/25434/files